_posts/2016-10-10-Help!-I'm-becoming-Post-Junior.md

```diff
@@ -0,0 +1,162 @@
+categories: [culture, juniors]
+series: Stages of Professional Growth
+---
+
+I’ve lived in NYC for 2 years now. This means I’ve been around long enough that some of the people I helped when they were learn to program have started to feel like they’re not “Juniors” anymore. 
+
+There is no arbitrary timeframe that you can say applies to that feeling of career progression, but from the explanations I’ve had recently - it’s been about feeling confident in your code, your impact and wanting to improve both.
+
+This post aims to be technology-agnostic, and if you sit somewhere at 1.5 -3 years of programming experience ( and most of that as writing production code ) then you’ll probably get something out of it. Though I’ll give some JS and iOS specific tips at the end.
+
+—
+
+This one is easy to start. At Artsy we have an [engineering ladder system][1] - which I’ll paraphrase below. It’s worth the full read though.
+
+Key point:
+
+> Performance evaluation at Artsy is composed of the what, _i.e. what you achieve that contributes to your team’s goals and ultimately Artsy goals_, and the how, _i.e. how you act and how those actions contribute to Artsy’s culture and values_.
+
+We measure your career stage by your **impact** at Artsy, this is initially on the products you’re working on, but eventually moves out to the culture and the business:
+
+- **Engineer 1** - Can ship a well defined product feature.
+- **Engineer 2** - Can independently own a product features, and can handle the communication with others around it.
+- **Engineer 3** - Can handle a suite of features, and broadly contribute within a domain. Can improve company culture.
+- **Engineer 4** - Can improve and be a multiplier on other people’s work, can anticipate larger trends and affect culture to avoid or steer in that direction.
+- **Engineer 5** - Defines technical culture, works on impacting all parts of our businesses and creates new opportunities for the company.
+
+What we’re talking about here is the transition from Engineer 1, to Engineer 2. Here’s our full unabridged description of an Engineer 2.
+
+>  ▪	Consistently writes and delivers correct and clean quality code with guidance.
+
+> ▪	Self-sufficient and makes steady progress on tasks.
+
+> ▪	Knows when to ask for help and how to get unblocked.
+
+> ▪	Makes steady, well-paced progress without the need for constant significant feedback from more senior engineers.
+
+> ▪	Owns a small-to-medium feature from technical design through completion.
+
+> ▪	Provides help and support outside of area under their responsibility.
+
+What can we gleam from this, that we start thinking of ways to improve ourselves as individual contributors?
+
+## Being Aware of How The Sausage Is Made
+
+It’s very unlikely that you are making all of the calls in a product, what is hopefully happening is that your team figures out a plan to ship something in a reasonable timeframe and as a team, you assign each other smaller talks that make that work. 
```

Comment:
Full stop after first sentence: 

It’s very unlikely that you are making all of the calls in a product. What is hopefully happening is that your team figures out a plan to ship something in a reasonable timeframe and as a team you assign each other smaller talks that make that work.